lib/kafo/configuration.rb

```diff
@@ -43,6 +43,7 @@ class Configuration
       ScenarioOption::KAFO_MODULES_DIR          => nil,
       ScenarioOption::CONFIG_HEADER_FILE        => nil,
       ScenarioOption::DONT_SAVE_ANSWERS         => nil,
+      ScenarioOption::CLASSES                   => {},
```

Comment:
The standard nomenclature we have been using is `modules`; but what we really seem to specify everywhere are puppet classes not puppet modules. Thus I decided to aim for using the correct representation.